host/src/server/api/v2/proof/prune.rs

```diff
@@ -0,0 +1,33 @@
+use axum::{debug_handler, extract::State, routing::post, Router};
+use raiko_task_manager::{get_task_manager, TaskManager};
+use utoipa::OpenApi;
+
+use crate::{interfaces::HostResult, server::api::v2::CancelStatus, ProverState};
+
+#[utoipa::path(post, path = "/proof/prune",
+    tag = "Proving",
+    responses (
+        (status = 200, description = "Successfully pruned tasks", body = CancelStatus)
+    )
+)]
+#[debug_handler(state = ProverState)]
+/// Prune all tasks.
+async fn prune_handler(State(prover_state): State<ProverState>) -> HostResult<CancelStatus> {
+    let mut manager = get_task_manager(&(&prover_state.opts).into());
+
+    manager.prune_db().await?;
+
+    Ok(CancelStatus::Ok)
```

Comment:
PruneStatus::Ok?

Reply:
Yes, just stubbed it because of the same structure